seed/challenges/00-getting-started/getting-started.json

```diff
@@ -85,6 +85,81 @@
       "tests": [],
       "type": "Waypoint",
       "challengeType": 7,
+      "titleId": "Mengetahui Bagaimana Free Code Camp bekerja",
+      "descriptionId": [
+	[
+	  "//i.imgur.com/6ibIavQ.jpg",
+	  "Gambaran 4 kelebihan Free Code Camp: Terhubung, Belajar JavaScript, Membangun Portofolio, Membantu nirlaba",
+	  "Selamat datang di Free Code Camp. Kami komunitas open source yang membantu kamu belajar code, kemudian berlatih membangun project untuk nirlaba.",
+	  ""
+	],
+	[
+	  "//i.imgur.com/Elb3dfj.jpg",
+	  "Screenshot beberapa campers coding bersama di Toronto.",
+	  "<bold>Belajar code sulit.</bold> Agar berhasil, kamu butuh banyak latihan dan dukungan. Itulah mengapa kami membangun kurikulum yang tepat dan dukungan komunitas.",
+	  ""
+	],
+	[
+	  "//i.imgur.com/WD3STY6.jpg",
+	  "Foto 3 campers yang mendapatkan pekerjaan setelah belajar code di Free Code Camp.",
+	  "Free Code Camp adalah jalan ampuh menjadi developer software. Banyak perusahaan mempekerjakan ribuan orang seperti kamu yang belajar code melalui Free Code Camp.",
+	  ""
+	],
+	[
+	  "//i.imgur.com/D7Y5luw.jpg",
+	  "Grafik rata rata pertumbuhan pekerjaan berdasarkan pertumbuhan lulusan ilmu komputer. Tersedia 1.4 juta pekerjaan dan hanya 400 ribu orang yang mengisi tempatnya.",
```

Comment:
The growth of jobs is compared to the growth of computer science graduates in the graph. I think _dibanding_ would be more accurate than _berdasarkan_

Change 1.4 to 1,4.